config/manager/manager.yaml

```diff
@@ -30,6 +30,7 @@ spec:
         args:
         - --leader-elect
         image: controller:latest
+        imagePullPolicy: Never
```

Comment:
I had some troubles to deploy the operator, maybe "IfNotPresent" would be preferable?

Reply:
This is reverted now